### DISCUSSION TITLE
Allow JP2 (JPEG-2000) files

### DISCUSSION THREAD

**Discussion by nemobis** [tjko/jpegoptim#93] [General]
`jpegoptim` works great for me; in my case I want to use the `-S` option to set a target size and downsize my JPEG file as needed. Some of my JPEG images are JPEG-2000 (JP2) files produced by ImageMagick (http://www.imagemagick.org/script/jp2.php ) and `jpegoptim` rejects them.

Is there any plan to make `jpegoptim` work with JP2 files too, at least for a subset of its functionality?

**Comment by mirabilos** [comment]
@nemobis the question was about JPEG files, not about JPEG2000 files. Convert those to JPEG first. (JPEG2000 decoders are currently in the process of being removed from a lot of software due to bugs, security holes, and lack of actual JPEG2000 images in the wild.)

From: http://stackoverflow.com/a/25360060/2171120

**Comment by nemobis** [comment]
I know the difference. My question is about JP2 files.

**Comment by ychevarrias** [comment]
jpegoptim actually support JP2?
